File: calculadora-java/src/main/java/org/example/model/operations/Subtraction.java
package org.example.model.operations;

public class Subtraction implements IOperation {
    private int id;

    public Subtraction() {
        this.id = 2;
    }

    public float operate (float valor1, float valor2) {
        return valor1 - valor2;
    }

    public int getId() {
        return id;
    }
}

File: calculadora-csharp/src/model/operations/Subtraction.cs
using System.Runtime.CompilerServices;

namespace calculadora_csharp.model.operations;

public class Subtraction : IOperation
{
    public int Id { get; set; } = 2;
    
    
    public float Operate(float num1, float num2)
    {
        return num1 - num2;
    }

}